public/admin/config.yml

```diff
@@ -124,6 +124,46 @@ collections:
           hint: "End time of event.",
           default: "",
         }
+      - {
+          label: "Location",
+          name: "location",
+          widget: "string",
+          required: false,
+          hint: "Location of event.",
+          default: "",
+        }
+      - {
+          label: "Event Details",
+          name: "eventDetails",
+          widget: "markdown",
+          required: false,
+          hint: "Details of the event, i.e. about the speakers, about the event host, etc..",
+          default: "",
+        }
+      - {
+          label: "Downloadable Media Content 1",
+          name: "downloadableMediaContent1",
+          widget: "file",
+          required: false,
```

Comment:
why not list -> https://www.netlifycms.org/docs/widgets/list/#list

Reply:
need `widget: "file"`

Reply:
ok updated to use list